desparchado/settings/base.py

```
@@ -241,10 +243,32 @@
                 },
-                "mapCenterLocationName": "Bogota",
             },
         },
     },
+    "Mapbox": {
+        "accessToken": MAPBOX_TOKEN,
+        "PointField": {
+            "interactive": {
+                "mapOptions": {
+                    "zoom": 12,
+                    "center": (-74.083655, 4.653411)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)